packages/base/components/Content/ContentRichText.vue

```diff
@@ -0,0 +1,66 @@
+<script setup>
+import { marked } from "marked";
+
+const props = defineProps({
+  text: {
+    type: String,
+    required: true,
+  },
+  // TODO: find a better and cleaner solution
```

Comment:
With a parent component? e.g.
```vue
<RichTextCard>
  <ContentRichText />
</RichTextCard>
```

Reply:
We don't need the rich text card variant for the DS website. And barely use it on the portal. It is only possibly used on feature pages. So instead I have removed the rich text card variant from here.